packages/apollo-collaboration-server/src/features/features.controller.ts

```diff
@@ -75,4 +77,15 @@ export class FeaturesController {
     this.logger.debug(`Get all features`)
     return this.featuresService.findAll()
   }
+
+  /**
+   * Updates end position of given feature. Before update, current end -position value is checked (against given old-value)
+   * @param serializedChange - Information containing featureId, newEndValue, oldEndValue
+   * @returns Return 'HttpStatus.OK' if featureId was found AND oldEndValue matched AND database update was successfull. Otherwise throw exception.
+   */
+  //  @UseGuards(JwtAuthGuard)
+  @Put('/updateEndPos')
+  updateMongo(@Body() serializedChange: SerializedChange) {
+    return this.featuresService.changeEndPos(serializedChange)
+  }
```

Comment:
This will need to be moved to `packages/apollo-collaboration-server/src/change/change.controller.ts`. Probably what needs to happen is that the login in `submitChange` in `change.controller.ts` needs to be moved to `change.service.ts`, and then we need to inject `featureModel` into the `ChangeService`.

Reply:
End-point has now been moved to change.controller.ts.